openmetadata-ui/src/main/resources/ui/src/axiosAPIs/teamsAPI.ts

```diff
@@ -102,11 +103,20 @@ export const deleteTeam = async (id: string) => {
   return response.data;
 };
 
-export const reactivateTeam = async (data: CreateTeam) => {
+export const changeTeamParent = async (data: CreateTeam) => {
   const response = await APIClient.put<CreateTeam, AxiosResponse<Team>>(
     '/teams',
     data
   );
 
   return response.data;
 };
```

Comment:
this API is for updating team in general right?

Reply:
@ShaileshParmar11  yes, it is.

Reply:
so we shoul name accordingly?

Reply:
@ShaileshParmar11  updated!